instagram/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+import "./App.css";
+// import SearchBar from "./components/SearchBar/SearchBar";
+// import PostsPage from "./components/PostContainer/PostsPage";
+import Auth from "./components/Authentication/Authenticate";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      loggedIn: false
```

Comment:
does this ever become True ?

Reply:
When there is a user in local storage it becomes true 
`if (localStorage.getItem("user")) {
      this.setState({
        loggedIn: true
      });
    } else {
      this.setState({
        loggedIn: false
      });
    }`